torch/optim/lr_scheduler.py

```diff
@@ -794,20 +810,20 @@ class CyclicLR(_LRScheduler):
     """
 
     def __init__(self,
-                 optimizer,
-                 base_lr,
-                 max_lr,
-                 step_size_up=2000,
-                 step_size_down=None,
-                 mode='triangular',
-                 gamma=1.,
-                 scale_fn=None,
-                 scale_mode='cycle',
-                 cycle_momentum=True,
-                 base_momentum=0.8,
-                 max_momentum=0.9,
-                 last_epoch=-1,
-                 verbose=False):
+                 optimizer: Optimizer,
```

Comment:
With `from typing import Type`, this should be `Type[Optimizer]`, right?

Reply:
I don't think so, the accepted input is an `Optimizer` instance, for that the annotation is correct. `Type` means the actual class object rather than an instance, see https://docs.python.org/3/library/typing.html#typing.Type